### DISCUSSION TITLE
I feel the interface has become very sluggish now

### DISCUSSION THREAD

**Discussion by Julianoe** [LycheeOrg/Lychee#2518] [Q&A] [answered]
I'm trying to update from version 4 to version 5 and the interface is very sluggish now. I'm on the same setup and nothing changed.  Like 2 seconds to display a menu or right click menus is kinda stupid. Is there anything that can be done or does the soft has just become heavier and then I just have to ditch it for something else?

**Comment by ildyria** [comment, marked as answer]
See here: https://github.com/LycheeOrg/Lychee/discussions/2485 :)

In the mean time, set up `LIVEWIRE_ENABLED=false`. Yes you will be losing a bit on the new features from the v5, but it will be faster.

**Reply by ildyria** [reply]
I have a PR that revamp the LycheeOrg website (which will look like this: https://lycheeorg.viguier.nl/

And about your question: https://lycheeorg.viguier.nl/2024-06-25-performance-problems/

**Reply by Julianoe** [reply]
curious to learn more, I'll read the post. Anyway, those technical choice dilemmas happen. I'll update the env variable and roll with it. I'll come back to you if notice something else. Thanks for the (very) fast answer! have a good one
